src/test/java/core/basesyntax/handlers/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.handlers;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private static OperationHandler purchaseOperationHandler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        purchaseOperationHandler = new PurchaseOperationHandler();
+    }
+
+    @Test
+    public void validOnePurchase_Ok() {
```

Comment:
Я думаю потрібно переназвати метод, наприклад так
```suggestion
    public void purchase_OneFruit_Ok() {
```